Libraries/NativeAnimation/Nodes/RCTTransformAnimatedNode.m

```diff
@@ -82,7 +82,12 @@ - (void)performUpdate
     }
   }
 
-  _propsDictionary[@"transform"] = [NSValue valueWithCATransform3D:transform];
+  _propsDictionary[@"transform"] = @[
```

Comment:
Could we just pass the transform config as is now and do the conversion to `CATransform3D` when setting the prop in the UIViewManager. This is what android does now, I think iOS still does some parsing in JS it could be a good opportunity to fix that as well. This would avoid having multiple representations of the transform (Array of objects -> CATransform -> number array -> CATransform). That way we'd always just have Array of objects -> CATransform.

See https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/processTransform.js#L36